### DISCUSSION TITLE
Ticker stopped working last night

### DISCUSSION THREAD

**Discussion by ngek** [achannarasappa/ticker#238] [General]
I have been running this for over a year without issue.  It stopped working late yesterday, 19-Jan-2023.  The header and footer appear but no data.

**Comment by achannarasappa** [comment]
Hey @ngek have you updated to the latest version (v4.5.5) released yesterday? The underlying Yahoo API has changed which broke ticker yesterday. v4.5.4 or lower will no longer work

**Reply by ngek** [reply]
Yes I did, but forgot to delete the old version that was in my local directory.  Working now. THANKS!
